### DISCUSSION TITLE
Issue Updating to 2.3.30 from 2.3.21

### DISCUSSION THREAD

**Discussion by ColeVan** [Security-Onion-Solutions/securityonion#3241] [Unsupported Versions] [answered]
After performing a sudo soup on Version 2.3.21 I received this error below. Any thoughts?
![image](https://user-images.githubusercontent.com/70167373/109719306-7ab56e00-7b76-11eb-93ef-c123d856e934.png)

**Comment by TOoSmOotH** [comment, marked as answer]
Try running soup again.

**Reply by ColeVan** [reply]
It worked
